Index: src/drivers/net/ethernet/qlogic/qede/Makefile
===================================================================
--- src.orig/drivers/net/ethernet/qlogic/qede/Makefile	2017-04-25 17:02:15.840664842 +0200
+++ src/drivers/net/ethernet/qlogic/qede/Makefile	2017-04-26 19:17:02.361321085 +0200
@@ -1,3 +1,6 @@
+CONFIG_QEDE := m
+CONFIG_QED_RDMA := y
+
 obj-$(CONFIG_QEDE) := qede.o
 
 qede-y := qede_main.o qede_fp.o qede_filter.o qede_ethtool.o
