Index: src/include/linux/qed/qed_backport_compat.h
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ src/include/linux/qed/qed_backport_compat.h	2017-04-25 17:01:34.589164789 +0200
@@ -0,0 +1,4 @@
+#ifndef _RH_INCLUDE_QED_BACKPORT_COMPAT_H_
+#define _RH_INCLUDE_QED_BACKPORT_COMPAT_H_
+
+#endif /* #ifndef _RH_INCLUDE_QED_BACKPORT_COMPAT_H_ */
Index: src/include/linux/qed/common_hsi.h
===================================================================
--- src.orig/include/linux/qed/common_hsi.h	2017-04-25 17:01:31.156206532 +0200
+++ src/include/linux/qed/common_hsi.h	2017-04-25 17:01:34.589164789 +0200
@@ -37,6 +37,8 @@
 #include <linux/bitops.h>
 #include <linux/slab.h>
 
+#include <linux/qed/qed_backport_compat.h>
+
 /* dma_addr_t manip */
 #define DMA_LO_LE(x)		cpu_to_le32(lower_32_bits(x))
 #define DMA_HI_LE(x)		cpu_to_le32(upper_32_bits(x))
Index: src/include/linux/qed/qede_roce.h
===================================================================
--- src.orig/include/linux/qed/qede_roce.h	2017-04-19 17:56:08.386757248 +0200
+++ src/include/linux/qed/qede_roce.h	2017-04-25 17:02:06.238780452 +0200
@@ -32,6 +32,9 @@
 #ifndef QEDE_ROCE_H
 #define QEDE_ROCE_H
 
+#include <linux/qed/qed_backport_compat.h>
+
+
 struct qedr_dev;
 struct qed_dev;
 struct qede_dev;
