Index: src/drivers/net/ethernet/qlogic/qede/qede.h
===================================================================
--- src.orig/drivers/net/ethernet/qlogic/qede/qede.h	2018-08-20 07:03:55.985509277 +0200
+++ src/drivers/net/ethernet/qlogic/qede/qede.h	2018-08-20 07:10:50.070554052 +0200
@@ -57,7 +57,7 @@
 #define DRV_MODULE_VERSION __stringify(QEDE_MAJOR_VERSION) "."	\
 		__stringify(QEDE_MINOR_VERSION) "."		\
 		__stringify(QEDE_REVISION_VERSION) "."		\
-		__stringify(QEDE_ENGINEERING_VERSION)
+		__stringify(QEDE_ENGINEERING_VERSION) "_dup7.6"
 
 #define DRV_MODULE_SYM		qede
 
