Index: src/include/linux/qed/qed_backport_compat.h
===================================================================
--- src.orig/include/linux/qed/qed_backport_compat.h	2017-04-20 18:46:44.333988050 +0200
+++ src/include/linux/qed/qed_backport_compat.h	2017-04-20 18:48:42.490579497 +0200
@@ -1,4 +1,17 @@
 #ifndef _RH_INCLUDE_QED_BACKPORT_COMPAT_H_
 #define _RH_INCLUDE_QED_BACKPORT_COMPAT_H_
 
+#define CONFIG_QED       1
+#define CONFIG_QED_FCOE  1
+#define CONFIG_QED_ISCSI 1
+#define CONFIG_QED_LL2   1
+#define CONFIG_QED_RDMA  1
+#define CONFIG_QED_SRIOV 1
+
+#define CONFIG_QEDE        1
+#define CONFIG_QEDE_GENEVE 1
+#define CONFIG_QEDE_RDMA   1
+#define CONFIG_QEDE_VXLAN  1
+
+
 #endif /* #ifndef _RH_INCLUDE_QED_BACKPORT_COMPAT_H_ */
