Index: src/drivers/net/ethernet/qlogic/qede/qede_main.c
===================================================================
--- src.orig/drivers/net/ethernet/qlogic/qede/qede_main.c	2017-04-20 18:54:40.839307597 +0200
+++ src/drivers/net/ethernet/qlogic/qede/qede_main.c	2017-04-20 19:00:17.677292127 +0200
@@ -47,12 +47,8 @@
 #include <linux/netdev_features.h>
 #include <linux/udp.h>
 #include <linux/tcp.h>
-#ifdef CONFIG_QEDE_VXLAN
 #include <net/vxlan.h>
-#endif
-#ifdef CONFIG_QEDE_GENEVE
 #include <net/geneve.h>
-#endif
 #include <linux/ip.h>
 #include <net/ipv6.h>
 #include <net/tcp.h>
